lib/beaker-pe/install/pe_utils.rb

```diff
@@ -289,7 +289,11 @@ def fetch_pe(hosts, opts)
         # @param [Host] host The host to install pacakges for
         # @api private
         def deploy_frictionless_to_master(host)
-          klass = host['platform'].gsub(/-/, '_').gsub(/\./,'')
+          platform = host['platform']
+          if platform == "aix-7.2-power"
```

Comment:
We don't have separate AIX 7.2 builds (they use 7.1). this means that when we classify the master for `pe_repo`, a 7.2 agent needs the 7.1 classification. The rest of the difference in the install are handled by `pe_repo` itself knowing how to install the 7.1 binaries on 7.2